gazelle/parser/comment.go

```diff
@@ -0,0 +1,98 @@
+package parser
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+)
+
+type CommentKind int
+
+const (
+	// BlockComment is a comment that starts with /*.
+	BlockComment CommentKind = iota
+	// LineComment is a comment that starts with //.
+	LineComment
+)
+
+// Comment is a token that represents either BlockComments or LineComments.
+type Comment struct {
+	// Kind represents the textual form of this comment. Either BlockComment or
+	// LineComment.
+	Kind CommentKind
+	// The text inside of the comment.
+	Value string
+}
```

Comment:
Why do you need to represent these as structs? Can't you just consume and discard them?

Reply:
Making a struct was a stylistic decision that allows for [type switching](https://tour.golang.org/methods/16) in the parser code.

Reply:
But why even return anything at all for a comment? Why not just treat it as another form of whitespace and discard it?

Reply:
+1 to not returning comments. I don't know whether whitespace is important in SASS, but it might not be necessary to return whitespace tokens either.